api/docs/docs/updates.md

```diff
@@ -7,6 +7,27 @@ description: Updates to the Covid Act Now API.
 
 Updates to the API will be reflected here.
 
+### CDC Community Level data now available 
+_Added on 2022-04-05_
+
+The CDC Community Level metric as well as the subcomponents of the CDC Community Level metric are now available within the Covid Act Now API. 
+
+[Learn more about how the CDC Community Level is measured](https://covidactnow.org/covid-risk-levels-metrics). 
+
+Fields added:
+* `communityLevels.cdcCommunityLevel`: The raw CDC Community Level metric. It serves as an exact mirror of the CDC’s published data, which is available for counties only and typically updates once a week.
+* `communityLevels.canCommunityLevel`: The Covid Act Now team’s version of CDC Community Level metric. Unlike the raw CDC community level, it is available for states and metros in addition to counties. It also uses different data sources in some cases (New York Times for cases, HHS for state hospitalization data). It updates daily, though county hospitalization data only updates once a week.
+* `metrics.weeklyNewCasesPer100k`: The number of new COVID cases per week per per 100K population.
+* `metrics.weeklyCovidAdmissionsPer100k`: The number of new COVID hospital admissions per week per 100K population. For counties this is calculated at the [Health Service Area level (HSA)](https://apidocs.covidactnow.org/data-definitions#health-service-areas).
```

Comment:
do we need to clarify (HSA) in each bullet?

Reply:
No strong feelings.  Honestly we could probably leave out "(HSA)" everywhere since we're spelling it out everywhere. 🤷 

That said, if we keep "(HSA)" I would move it before "level" (here and below).

Reply:
Yeah true--good call--I've opted to remove them entirely in c4d756d1d0aee2e3cf650c072ad41554294769e1